def.bzl

```diff
@@ -18,9 +18,11 @@ load(
     _go_rule = "go_rule",
 )
 load("@bazel_skylib//lib:shell.bzl", "shell")
-load("//internal:go_repository.bzl", "go_repository")
+load("//internal:go_repository.bzl", _go_repository = "go_repository")
 load("//internal:overlay_repository.bzl", "git_repository", "http_archive")
```

Comment:
We'll also need to re-export `git_repository` and `http_archive` from here and deps.bzl below.